### PR TITLE
[iOS] Fix Update iOS Snapshots CI with PR

### LINKED ISSUE #534: [iOS] Update Snapshots with CI is failed.
**Overview (Required)**
- Because `main` is protected branch, so the CI failed.

**Steps To Reproduce**

- Merge a branch that includes new snapshots.

**Expected behavior**

- After above branch merged, snapshots are updated by CI.

**Additional context**

- Change to process that opens PR is maybe better.
- During not fixed this, please ignore snapshot-testing CI failing state.

### PR DESCRIPTION
**Pull request by ry-itto**
## Issue
- close #534 

## Overview (Required)
- Use PullRequest instead of push to `main` branch.
- I tried this at https://github.com/ry-itto-playground/test-fork-commit-with-action/commit/1bcdd745d2ae588ed7440ceea23a7e6273c02db1

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: .github/workflows/ios-commit-snapshots.yml [hunk 1]
```diff
@@ -35,10 +35,12 @@ jobs:
...
-        uses: stefanzweifel/git-auto-commit-action@v4
+      - name: Create Update Snapshots PullRequest
+        uses: peter-evans/create-pull-request@v3
         with:
-          commit_message: Update iOS Snapshots
-          branch: 'main'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          commit-message: Update iOS Snapshots
```
Comment (ostk0069) [inline .github/workflows/ios-commit-snapshots.yml:41]:
```suggestion
          commit-message: [iOS] Update iOS Snapshots
```

**Review by ostk0069 — COMMENTED** [review]
Please update the README as well.

**Comment by ostk0069** [comment]
@ry-itto 
Let me check about how we manage the created PR.
Is it okay either me or you who is free to do will review and merge (and sometime fix) the created PR?
If the current PR was made by me or you, who will be the best person to review and merge (and sometime fix)?

(念の為日本語でも)
snapshotの更新のPRに対しての運用について確認させてください
- PRが私かry-ittoさんによって作成された場合はどのように運用するのがいいでしょうか
- PRが私かry-ittoさん以外の人によって作成された場合は誰がmergeしたのかに限らず、手が空いてる方がレビューとmerge(時には修正)をするのがいいと思っていますがどうでしょうか

**Comment by ry-itto** [comment]
> Is it okay either me or you who is free to do will review and merge (and sometime fix) the created PR?

Yes, it is okey. Let's do it by either person who has time.

> If the current PR was made by me or you, who will be the best person to review and merge (and sometime fix)?

Let the another person do it. (If I made PR, you should review created PR.)

**Comment by ostk0069** [comment]
> Let the another person do it. (If I made PR, you should review created PR.)

Sounds good 👍

**Review by ostk0069 — APPROVED** [review]

**Pull request merged**